README.md

```diff
@@ -28,130 +28,6 @@ $ yarn
 $ yarn start
 ```
 
-### Directory Structure
```

Comment:
Given the amount of changes made to the website, this section becomes obsolete real quick, so we could probably just remove it.